.repo-templates/pubsub/README.md

```diff
@@ -0,0 +1,36 @@
+# Watermill {{ .HumanName }}
+<img align="right" width="200" src="https://threedots.tech/watermill-io/watermill-logo.png">
+
+[![CircleCI](https://circleci.com/gh/ThreeDotsLabs/{{ .Name }}/tree/master.svg?style=svg)](https://circleci.com/gh/ThreeDotsLabs/{{ .Name }}/tree/master)
+[![Go Report Card](https://goreportcard.com/badge/github.com/ThreeDotsLabs/{{ .Name }})](https://goreportcard.com/report/github.com/ThreeDotsLabs/{{ .Name }})
+
+This is Pub/Sub for the [Watermill](https://watermill.io/) project.
+
+All Pub/Sub implementations can be found in [https://watermill.io/docs/pub-sub-implementations/](https://watermill.io/docs/pub-sub-implementations/).
+
+**Migration to separated Pub/Sub repositories in progress**.
+
+Watermill is a Go library for working efficiently with message streams. It is intended
+for building event driven applications, enabling event sourcing, RPC over messages,
+sagas and basically whatever else comes to your mind. You can use conventional pub/sub
+implementations like Kafka or RabbitMQ, but also HTTP or MySQL binlog if that fits your use case.
+
+Documentation: https://watermill.io/
+
+Getting started guide: https://watermill.io/docs/getting-started/
+
+## Contributing
+
+All contributions are very much welcome. If you'd like to help with Watermill development,
+please see [open issues](https://github.com/ThreeDotsLabs/watermill/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+)
```

Comment:
there's also open issues for subrepositories. Do we want to link to both? or just the subrepo?

Reply:
we agreed to keep the issues in the main repo, to avoid distributing them